### PR TITLE
Add two methods to the modifiable GMPE

### PR DESCRIPTION
**Pull request by mmpagani**
This PR adds to methods to the modifiable GMPE that help in to define the total standard deviation in case of site specific studies. Note that the same behaviour can be achieved by extending more traditional GMPEs with a different table of coefficients.

### CODE REVIEW

File: openquake/hazardlib/tests/gsim/mgmpe/modifiable_gmpe_test.py [hunk 1]
```diff
@@ -175,6 +175,38 @@ def test_fixed_total_sigma(self):
...
+
+        gmm = ModifiableGMPE(
+            gmpe={gmpe_name: {}},
+            add_delta_std_to_total_std={"delta": -0.20})
+        imt = PGA()
+        [stddev] = gmm.get_mean_and_stddevs(self.sites, self.rup,
+                                            self.dists, imt, stddevs)[1]
+
+        # Original total std for PGA is 0.6201
+        np.testing.assert_almost_equal(stddev[0], 0.68344277, decimal=6)
```
[the remark this answers is not in the document]
Comment (mmpagani) [reply openquake/hazardlib/tests/gsim/mgmpe/modifiable_gmpe_test.py:191]: Done

File: openquake/hazardlib/tests/gsim/mgmpe/modifiable_gmpe_test.py [hunk 2]
```diff
@@ -175,6 +175,38 @@ def test_fixed_total_sigma(self):
...
+        stddevs = [const.StdDev.TOTAL]
+
+        gmm = ModifiableGMPE(
+            gmpe={gmpe_name: {}},
+            set_total_std_as_tau_plus_delta={"delta": 0.45})
+        imt = PGA()
+        [stddev] = gmm.get_mean_and_stddevs(self.sites, self.rup,
+                                            self.dists, imt, stddevs)[1]
+
+        # Original total std for PGA is 0.6201
```
[the remark this answers is not in the document]
Comment (mmpagani) [reply openquake/hazardlib/tests/gsim/mgmpe/modifiable_gmpe_test.py]: Fixing copy and pasting problem. Done

File: openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py [hunk 3]
```diff
@@ -82,7 +82,8 @@ def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
         """
         working_std_types = copy.copy(stddev_types)
 
-        if 'set_between_epsilon' in self.params:
+        if ('set_between_epsilon' in self.params or
```
[the remark this answers is not in the document]
Comment (mmpagani) [reply openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py:85]: That's another function not part of this PR

**Comment by rcgee** [comment]
LGTM

**Review by micheles — APPROVED** [review]

**Pull request merged**